the-collab-lab/tcl-77-smart-shopping-list
# Header Design & Carousel Base + About page

## Description



Moving the navigation up to the top position and adding our design to the links, header and the sign out button. It should change to a hamburger on smaller screens and still work to navigate through the app. Added the sigma design to the about page and got the skeleton of the carousel we will need the pictures before it can be completed.

## Related Issue


Closes #53 , #54 , #42 

## Acceptance Criteria


- [x] Get the navigation/header styled based on design
- [x] Make sure still works as it did
- [x] Get the About page to look like design
- [x] Create a base Carousel that needs images when design is done 

## Type of Changes

|        | Type |
|--------|--------|
|  ✓  | 💄 UI Updates |
|  ✓  | ✨ Enhancements | 



## Updates

### Before


<img width="1624" alt="Screenshot 2024-10-09 at 4 50 35 AM" src="https://github.com/user-attachments/assets/e1623e3f-5363-4f5d-9f3c-d810d538208d">

<img width="1624" alt="Screenshot 2024-10-09 at 4 50 42 AM" src="https://github.com/user-attachments/assets/cb244415-b276-4f9d-8276-608923a04f98">

<img width="1607" alt="Screenshot 2024-10-10 at 3 47 49 AM" src="https://github.com/user-attachments/assets/47c0eaed-b825-45f6-bbf1-19963164439e">

### After


<img width="1624" alt="Screenshot 2024-10-09 at 4 50 49 AM" src="https://github.com/user-attachments/assets/fcc28662-cf8d-481b-8366-55bb00d102c4">

<img width="1624" alt="Screenshot 2024-10-09 at 4 50 54 AM" src="https://github.com/user-attachments/assets/bd31d80b-2328-4420-a25d-44a94921dd95">

<img width="1607" alt="Screenshot 2024-10-10 at 3 47 34 AM" src="https://github.com/user-attachments/assets/397c3552-8544-444b-ad20-1f26f3b2b7f2">


## Testing Steps / QA Criteria



- Click the preview link in the PR
- Make sure all the buttons of the nav bar at the top still take you to the spots that are needed
- Open the dev tools and test a phone between 430px - 412px media points should have the hamburger menu
- make sure in mobile view can still navigate
